manuals/manual_convert_schedule_to_events.md

```diff
@@ -0,0 +1,463 @@
+<h3 style="color: #f46242;"><strong>Warning</strong>: <i>the specified events below are not yet implemented within Pronto yet and thus cannot be processed</i></h3>
+
+# Convert carrier schedule to port call events
+We frequently receive the question from carriers how they can share their schedule with Pronto.
+
+This manual contains instructions on how to do this.
+
+## Example carrier schedule
+For this manual we take a fictitious port "Gilo" with UNLOCODE `XXGIL`. Gilo has a single terminal managing the ports 3 berth `Gilo 1`, `Gilo 2` and `Gilo 3`. Our 
+
+At 2018-01-01 14:00 the latest schedule for the carrier as kept in its Carrier Schedule System (CSS) is as follows:
```

Comment:
Does CGA CGM, Maersk & co call it a CSS themselves too?

Reply:
I've checked with Rob, he knowns Maersk calls it GSIS, he was not sure about CGA CGM.

Reply:
Then I would just refer to it as carrier scheduling system and skip the abbreviation